tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -253,10 +253,11 @@ def run(self):
                     ).load_all() or []
                     volumes = []
                     for volume in all_volumes:
-                        if not str(volume.deleted).lower() == "true" or \
-                            volume.current_job.get('status', '') \
-                            in ['', 'finished', 'failed']:
-                            volumes.append(volume)
+                        if volume.exists():
```

Comment:
all_volumes we are getting for etcd only. How come few of them doesnt exist and why we need this check?

Reply:
that's correct, I think I misunderstood the logic here, i will remove this